app/code/Magento/Inventory/Model/GetProductQuantityInStock.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model;
+
+use Magento\Inventory\Model\ResourceModel\Reservation\ReservationQuantity;
+use Magento\Inventory\Model\ResourceModel\Stock\StockItemQuantity;
+use Magento\InventoryApi\Api\GetProductQuantityInStockInterface;
+
+/**
+ * Return Quantity of products available to be sold by Product SKU and Stock Id
+ *
+ * @see \Magento\InventoryApi\Api\GetProductQuantityInStockInterface
+ * @api
+ */
+class GetProductQuantityInStock implements GetProductQuantityInStockInterface
```

Comment:
This service looks like a perfect candidate to be covered with both:
- Integration 
and
- Web API 
testing

Reply:
As proposed by @naydav and discussed with @maghamed we will simplify the implementation getting rid of the command layer; 
thus we will delegate effective computation to resource model, injected into the services in place of the command.